Simplenote/Classes/NoteInfoViewController.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  Created by Kevin LaCoste on 2020-05-15.
+//  Copyright © 2020 Automattic. All rights reserved.
+//
+
+import UIKit
+
+class NoteInfoViewController: UIViewController, UITableViewDataSource {
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var dismissButton: UIButton!
+    @IBOutlet weak var tablewView: UITableView!
+    
+    @objc var dismissAction: (() -> Void)? = nil
+    @objc var note: Note?
+    @objc var hasSafeAreas = false
+    
+    var cells = [UITableViewCell]()
+    
+    // MARK: - View Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setUpTitleLabel()
+        setUpTableCells()
+        
+        if dismissAction == nil {
+            dismissButton.isHidden = true
+        }
```

Comment:
Update: I've realized that since SPActionSheet is not using ViewController containment, `viewWillAppear` will never get called. Good to keep this one in `viewDidLoad` (we probably wanna drop SPActionSheet!)

Reply:
Agreed on eventually removing SPActionSheet.